components/TrainList.js

```diff
@@ -54,6 +53,7 @@ const TrainList = ({ trains }) => {
 
 const styles = StyleSheet.create({
   container: {
+    marginTop: 10
```

Comment:
Missing trailing comma.

[Read more about it here.](http://eslint.org/docs/rules/comma-dangle)